brainstormforce/astra
# AST-3157 - Filter to convert Line-height unit to desired unit.

### Description

- Before Astra 4.0.0, we were not adding any unit to the body line height: [Markup 2023-01-20 at 13.05.31.png](https://d.pr/i/OdBHXi) . But now, we have started adding a unit (em) to the body line-height causing problems for the users.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3157

### Screenshots

https://d.pr/v/SFv7Ka

### Types of changes




Fixes bug and also an improvement to change unit as per user needs.

### How has this been tested?

- Check the Line-height and see it works as expected like before with the default em/px units.
- Now Add the new filter and check if Line-height applies with the filter provided unit value as expected.
- Check if it works as expected for different Typography Line-height settings.
- Check that filter does not change the unit for other Font Extras.
### Checklist:
